Tests/PostgresNIOTests/New/Data/String+PSQLCodableTests.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 03.02.21.
+//
+
+import XCTest
```

Comment:
Ditch Xcode template headers.
```suggestion
import XCTest
```